wordpress/README.md

```diff
@@ -0,0 +1,280 @@
+# Quick Start
+
+This repository includes the necessary files to package and deploy [Wordpress](https://www.wordpress.org/), a open source content management system, into a private cloud environment using [Gravity](http://gravitational.com/gravity/).  
```

Comment:
> into a private cloud environment

Why cloud? It can be on-prem too.

Reply:
modified to on-prem or public/private cloud